src/openrct2-ui/windows/Cheats.cpp

```diff
@@ -819,108 +826,82 @@ static void window_cheats_guests_mouseup(rct_window* w, rct_widgetindex widgetIn
             window_cheats_set_page(w, widgetIndex - WIDX_TAB_1);
             break;
         case WIDX_GUEST_HAPPINESS_MAX:
-            game_do_command(
-                0, GAME_COMMAND_FLAG_APPLY, CHEAT_SETGUESTPARAMETER, GUEST_PARAMETER_HAPPINESS, GAME_COMMAND_CHEAT,
-                PEEP_MAX_HAPPINESS, 0);
+            CheatsSet(CheatType::SetGuestParameter, GUEST_PARAMETER_HAPPINESS, PEEP_MAX_HAPPINESS);
             break;
         case WIDX_GUEST_HAPPINESS_MIN:
-            game_do_command(
-                0, GAME_COMMAND_FLAG_APPLY, CHEAT_SETGUESTPARAMETER, GUEST_PARAMETER_HAPPINESS, GAME_COMMAND_CHEAT, 0, 0);
+            CheatsSet(CheatType::SetGuestParameter, GUEST_PARAMETER_HAPPINESS, 0);
             break;
         case WIDX_GUEST_ENERGY_MAX:
-            game_do_command(
-                0, GAME_COMMAND_FLAG_APPLY, CHEAT_SETGUESTPARAMETER, GUEST_PARAMETER_ENERGY, GAME_COMMAND_CHEAT,
-                PEEP_MAX_ENERGY, 0);
+            CheatsSet(CheatType::SetGuestParameter, GUEST_PARAMETER_ENERGY, PEEP_MAX_ENERGY);
             break;
         case WIDX_GUEST_ENERGY_MIN:
-            game_do_command(
-                0, GAME_COMMAND_FLAG_APPLY, CHEAT_SETGUESTPARAMETER, GUEST_PARAMETER_ENERGY, GAME_COMMAND_CHEAT,
-                PEEP_MIN_ENERGY, 0);
+            CheatsSet(CheatType::SetGuestParameter, GUEST_PARAMETER_ENERGY, PEEP_MIN_ENERGY);
             break;
         case WIDX_GUEST_HUNGER_MAX:
-            game_do_command(
-                0, GAME_COMMAND_FLAG_APPLY, CHEAT_SETGUESTPARAMETER, GUEST_PARAMETER_HUNGER, GAME_COMMAND_CHEAT, 0, 0);
+            CheatsSet(CheatType::SetGuestParameter, GUEST_PARAMETER_HUNGER, 0);
             break;
         case WIDX_GUEST_HUNGER_MIN:
-            game_do_command(
-                0, GAME_COMMAND_FLAG_APPLY, CHEAT_SETGUESTPARAMETER, GUEST_PARAMETER_HUNGER, GAME_COMMAND_CHEAT, 255, 0);
+            CheatsSet(CheatType::SetGuestParameter, GUEST_PARAMETER_HUNGER, 255);
```

Comment:
If there is a constant defined for max hunger, it should be used. Otherwise, it should be added. Same applies to other values here.